src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const shortTerm = 3;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const price = 40;
+
+  if (days >= longTerm) {
+    return days * price - bigDiscount;
+  }
```

Comment:
The condition `if (days >= longTerm)` correctly checks for long-term rentals. Ensure that the discount logic aligns with the task requirements.